ieeeuoft/hackathon-template
# IEEE-126-Convert-TeamCard-to-TypeScript and IEEE-127-Connect-TeamCard-on-dashboard-to-redux-store

## Overview

- Resolves IEEE-126 and IEEE-127
- Converted TeamCard.js, TeamCard.test.js, Team.js, Team.test.js to TypeScript
- Connected TeamCard on Dashboard to redux store

## Unit Tests Created

- check to see team member names and team code is rendered


## Steps to QA

- go to the dashboard
- make sure you have a team
- add users to your team --> notice their names will show up
